content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -150,7 +123,7 @@ You can nest `include` options to include relations of relations. The following
 <cmd>
 
 ```ts
-const user = await prisma.user.findMany({
+const user = await prisma.user.findFirst({
```

Comment:
This page is inconsistent between `getModel` vs `model`. I'm ok with either but we should be consistent.